src/components/record/MiniView.jsx

```diff
@@ -19,6 +19,14 @@ const propTypes = {
   readRecord: PropTypes.func,
 };
 
+const messages = defineMessages({
+  authority: {
+    id: 'recordTitleBar.authority',
```

Comment:
The convention I've been using is that the message id starts with the name of the component.
```suggestion
    id: 'miniView.authority',
```